cargo-dist/templates/installer/installer.ps1.j2

```diff
@@ -381,7 +381,8 @@ function Invoke-Installer($artifacts, $platforms) {
         Write-Information ""
         Write-Information "To add $dest_dir to your PATH, either restart your system or run:"
         Write-Information ""
-        Write-Information "    `$env:Path = `"$dest_dir;`$env:Path`""
+        Write-Information "    `set Path=$dest_dir;%Path%   (cmd)"
```

Comment:
Is the backtick the escape character? If so, do we actually need to escape the start of `set`?

Reply:
wow i can't believe you caught that by eye (you are correct and fixed)

Reply:
Turns out I'm hyper-sensitive to mismatched backticks from bash land 😅